api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -36,6 +36,8 @@
   private static final String CLUSTER_LABEL_AOU = "all-of-us";
   private static final String CLUSTER_LABEL_CREATED_BY = "created-by";
   private static final String WORKSPACE_CDR = "WORKSPACE_CDR";
+  private static final String JUPYTER_DOCKER_IMAGE =
```

Comment:
nit: I can see this being a value that should be on our WorkbenchConfig. It would let us change the docker image without deploys if needed and there's also the benefit of making it easier to find. I can see this being a value that the researcher team or support team would be interested in knowing.

Reply:
+1 We may even eventually want to support multiple images that we decide among when launching a cluster based on user, free tier vs billed, etc.